messaging/src/main/java/org/axonframework/eventhandling/TrackingEventProcessor.java

```diff
@@ -350,8 +350,12 @@ private void releaseToken(Segment segment) {
         try {
             transactionManager.executeInTransaction(() -> tokenStore.releaseClaim(getName(), segment.getSegmentId()));
             logger.info("Released claim");
-        } catch (Exception e) {
-            logger.info("Release claim failed", e);
+        } catch (Exception e) {  
+        	if (logger.isDebugEnabled()) {
```

Comment:
Nit: The indentation doesn't seem aligned with Axon Framework's default.

Reply:
Can you please provide a commit suggestion for this? I am using Eclipse and cannot use the IntelliJ settings.